lib/linguist/heuristics.rb

```diff
@@ -105,6 +105,12 @@ def call(data)
       end
     end
 
+    disambiguate ".auradoc" do |data|
```

Comment:
Heuristic rules are only required for conflicting languages. You can check in `languages.yml` and remove heuristic rules for extensions that are only used by Lightning Component.